privacyidea/lib/smsprovider/FirebaseProvider.py

```diff
@@ -250,6 +257,14 @@ def parameters(cls):
                           "required": True,
                           "description": _("The filename of the JSON config file, that allows privacyIDEA to talk"
                                            " to the Firebase REST API.")
+                      },
+                      FIREBASE_CONFIG.HTTP_PROXY: {
```

Comment:
This setting is not necessary, since the firebaseprovider always connects via HTTPS.